toolkit/scripts/toolchain/container/Dockerfile

```diff
@@ -63,7 +63,7 @@ RUN wget -nv --no-clobber --timeout=30 --continue --input-file=$LFS/tools/toolch
 # Disable downloading from remote sources by default. The 'toolchain-local-wget-list' generated for the above line will download from $(SOURCE_URL)
 # The 'toolchain-remote-wget-list' is still available and can be used as an alternate to $(SOURCE_URL) if desired.
 #RUN wget -nv --no-clobber --timeout=30 --continue --input-file=$LFS/tools/toolchain-remote-wget-list --directory-prefix=$LFS/sources; exit 0
-RUN wget -nv --no-clobber --timeout=30 --continue https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner-3/6.6.104.2.tar.gz -O kernel-6.6.104.2.tar.gz --directory-prefix=$LFS/sources; exit 0
+#RUN wget -nv --no-clobber --timeout=30 --continue https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner-3/6.6.104.2.tar.gz -O kernel-6.6.104.2.tar.gz --directory-prefix=$LFS/sources; exit 0
```

Comment:
I'm commenting this out for now, since the kernel source is downloaded in a previous step, and the file is getting truncated due to a wget2 bug.
I'm not deleting this line completely yet, since we have automation that updates this line, and that should be changed at the same time. Will follow up with a separate PR for that.